packages/testring-recorder-frontend/src/index.tsx

```diff
@@ -1,8 +1,9 @@
+import * as React from 'react';
```

Comment:
In this package TS runs with flag allowSyntheticDefaultImports.
@johnthecat can't we import it just like `import React from 'react';`